### PR TITLE
chore(blog): write in in english

### PR DESCRIPTION
**Pull request by AugustinMauroy**

### CODE REVIEW

**Comment by AugustinMauroy** [comment]
@SamTV12345 I invite you to read/correct it before posting it in public. And you are welcome to co-write

File: content/posts/etherpad-next.en.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Rewrite of etherpad-lite with next.js
+description: This project is a rewrite of etherpad-lite with next.js and tailwindcss.
+date: 2024-03-21
+authors: github:AugustinMauroy
+---
+
+> **Note:** This project under development and not ready for production.
```
Comment (SamTV12345) [inline content/posts/etherpad-next.en.mdx]: This project is under development and not ready for production

File: content/posts/etherpad-next.en.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+> So take attention when this post is published to understand the current state of the project.
+
+## What is Etherpad-lite?
+
+Etherpad is a highly customizable Open Source online editor providing collaborative editing in real-time. It is a web-based editor providing collaborative editing in real-time. It is a web-based editor that allows multiple people to write simultaneously in the same document.
```
Comment (SamTV12345) [inline content/posts/etherpad-next.en.mdx]: It is a web-based editor featuring collaborative editing in real-time

File: content/posts/etherpad-next.en.mdx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## How it's actually working?
+
+Etherpad-lite is a Node.js application that uses the `express` framework. It uses the `socket.io` library to handle real-time communication between clients and the server. The front-end is built with `jQuery`.
+
+## Why a rewrite?
+
+On Etherpad-lite V2, [@SamTV12345](https://github.com/SamTV12345) rewrote the back-end in typescript and fix some bugs **congratulations to him 🎉**. But the front-end is still in jQuery and the codebase is not easy to maintain.
```
Comment (SamTV12345) [inline content/posts/etherpad-next.en.mdx]: fixed

File: content/posts/etherpad-next.en.mdx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+I saw the typescript pr and I thought it was a good opportunity help. So I helped him with reviewing the code.
+
+After that [Sam](https://github.com/SamTV12345) contact me to propose me to rewrite the front-end in next.js. I accepted because I love next.js and I love Open Source.
```
Comment (SamTV12345) [inline content/posts/etherpad-next.en.mdx]: contacted

File: content/posts/etherpad-next.en.mdx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+### What I want to keep backward compatibility?
+
+- Multiple db support for the project. But now Prisma isn't supporting multiple db. So we will have to find a solution. And for moment we only support MySQL.
+- Plugins support. We will try to keep the same plugin system but we didn't know if it's possible.
+- Moodle integration. Moodle is a popular LMS. We will try to keep the same integration but we didn't know if it's possible.
+
+## OpenSSF Scorecard
+
+We will try to have a good score on the OpenSSF Scorecard. It's important to have a secure project. We will try to have a good score.
+It's also give security overview of the project.
```
Comment (SamTV12345) [inline content/posts/etherpad-next.en.mdx]: It also proposes a security overview of the project

**Review by SamTV12345 — APPROVED** [review]

**Comment by SamTV12345** [comment]
Great article :). Thanks for giving compliments.

**Comment by AugustinMauroy** [comment]
Thanks for you review I'm little bit dyslexic

**Comment by SamTV12345** [comment]
No worries. Great writing style. I like the article ;)

**Pull request merged**